README.md

```diff
@@ -134,6 +134,9 @@ For `labelRenderer`, you can provide a full path - [see this PR](https://github.
 - `shouldExpandNode: function(keyName, data, level)` - determines if node should be expanded (root is expanded by default)
 - `hideRoot: Boolean` - if `true`, the root node is hidden.
 - `sortObjectKeys: Boolean | function(a, b)` - sorts object keys with compare function (optional). Isn't applied to iterable maps like `Immutable.Map`.
+- `onNodeExpansionChanging: function(keyName, data, level, expanded)` - invoked when a node is expanding or collapsing.
+- `onNodeExpansionChanged: function(keyName, data, level, expanded)` - invoked when a node is expanded or collapsed.
```

Comment:
Then my suggestion is this:
- rename `onNodeExpansionChanging` to `onExpansionArrowClicked` (or something like that)
- move `onNodeExpansionChanged` to `componentDidUpdate` (with checking `this.state.expanded !== prevState.expanded`), so it would be called every time `expanded` was changed, not just when user clicked it - I feel like it would be more consistent